spec.txt

```diff
@@ -6343,6 +6342,21 @@ Unicode nonbreaking spaces count as whitespace, too:
 ````````````````````````````````
 
 
+Unicode symbols count as punctuation, too:
```

Comment:
the position of the case and this line are due to the test before it, which reads “Unicode nonbreaking spaces count as whitespace, too:”